### ISSUE TITLE
ADI ladder / tapes black under OpenGL mode

### ISSUE THREAD

**Issue by MOARdV** [Mihara/RasterPropMonitor#155]
http://forum.kerbalspaceprogram.com/threads/57603-0-25-RasterPropMonitor-putting-the-A-in-your-IVA-%28v0-18-3%29-8-Oct?p=1737558&viewfull=1#post1737558

I haven't looked into this, but it's cropped up in the RPM thread from time to time.  May as well get it documented in the issue list.

**Comment by MOARdV** [comment]
This is complicated by my inability to run KSP in OpenGL mode since 0.24 - when I start with -force-opengl, the only part of the screen that renders is the lower-left quadrant.  The rest is blanked out, which makes it virtually impossible to load a craft and launch a flight.

**Comment by MOARdV** [comment]
Okay, the workaround to get OpenGL working again is to alt-tab on the main menu, and then go back into the game.  I see the black texture problem in the HUD ladder, although I see white artifacts if I use DX instead.  I'll see if changing the shader from Hidden/GUI-Texture helps.  Worst case, I can refactor the HUD design and use a custom shader, since I finally figured out how to do that.

**Comment by MOARdV** [comment]
I looked at this some more today.  I do not understand what's going on, and why there are artifacts with the HUD ladder.  I've played with a few theories:

1) It's because the ladder / overlay have gray backgrounds, and there's a leak or "dark color" due to an alpha blending quirk in OpenGL mode.  Except if I redo them with green (low alpha) backgrounds, the artifacts are still there.

2) It's because the HUD is drawing on _MainTex, not an emissive layer.  I have no way to test this, since the HUD model doesn't have an _Emissive layer (or it has a different name? or I didn't set something else up right?).

3) It's an artifact of point sampling the texture.  Maybe the problem is that it's the texture blit showing point sampling artifacts because the texture is larger than the rendered area on screen.  I tried setting useMips and generateMips to true in the rentex, and that seemed subjectively to help in DirectX, but it rendered nothing in OGL - I may need to get more elaborate and render to the rentex and blit it into a conventional texture, and generate mipmaps on that, instead.
